packages/mobx-state-tree/src/core/node/scalar-node.ts

```diff
@@ -70,6 +76,10 @@ export class ScalarNode extends BaseNode {
     }
 
     get value(): any {
+        // make sure the parent chain is created when this is accessed
+        if (this.parent) {
```

Comment:
In theory no, but in theory there was no way to access an object node until resolving references came along. I put it "just in case".
I'll convert it to a comment
```
        // if we ever find a way where scalar nodes can be accessed without iterating through its parent
        // uncomment this to make sure the parent chain is created when this is accessed
        // if (this.parent) {
        //     this.parent.createObservableInstanceIfNeeded()
        // }
```

Reply:
commented out